utils/devServerProxy/config/devAccounts.config.js

```diff
@@ -16,14 +16,14 @@ export default {
     DEV0000USLTMQ: ['US', ['long_term'], 'pay_monthly_lt_mq'],
     DEV00USLTMQEZ: ['US', ['long_term'], 'pay_monthly_lt_mqez'],
     DEV00USLTMQRB: ['US', ['long_term'], 'pay_monthly_lt_mqez_rb'],
-    DEV0USLTNQMAX: ['US', ['long_term'], 'pay_monthly_lt_nq_max'],
-    DEV0USLTNQMIN: ['US', ['long_term'], 'pay_monthly_lt_nq_min'],
-    DEVUSLTNQEZMX: ['US', ['long_term'], 'pay_monthly_lt_nqez_max'],
-    DEVUSLTNQEZMN: ['US', ['long_term'], 'pay_monthly_lt_nqez_min'],
+    DEV00USLTNQEZ: ['US', ['long_term'], 'pay_monthly_lt_nqez'],
+    DEV0000USLTNQ: ['US', ['long_term'], 'pay_monthly_lt_nq'],
     DEV0000USLTSQ: ['US', ['long_term'], 'pay_monthly_lt_sq'],
     DEV00USLTSQEZ: ['US', ['long_term'], 'pay_monthly_lt_sqez'],
     DEV00USLTSQRB: ['US', ['long_term'], 'pay_monthly_lt_sqez_rb'],
 
+    DEV0USLTNAMNT: ['US', ['long_term'], 'pay_monthly_lt_no_amnt'],
```

Comment:
```suggestion
    DEVUSLTNOAMNT: ['US', ['long_term'], 'pay_monthly_lt_no_amnt'],
```
Small change to the account name to match what's in the `demo/helpers/accounts.js` file.